chainer/functions/evaluation/accuracy.py

```diff
@@ -25,22 +28,42 @@ def forward(self, inputs):
         xp = cuda.get_array_module(*inputs)
         y, t = inputs
         y = y.reshape(len(y), -1)  # flatten
-        pred = y.argmax(axis=1)
-        return xp.asarray((pred == t).mean(dtype='f')),
+
+        if self.ignore_label is not None:
+            mask = (t == self.ignore_label)
+            ignore_cnt = mask.sum()
+
+            # will always be true when the true label is ignore_label
+            # TODO(someone)
```

Comment:
Please write your name or your account name. This name is not who need todo, but who wrote the TODO. *We* will fix it.